bootcamp-hub/values.yaml

```diff
@@ -51,6 +45,13 @@ jupyterhub:
       users:
         - earth-lab
         - jenp0277
+        - betatim
+        - lwasser
```

Comment:
Without adding myself to the whitelist here I can't login. This is because we use a organisation whitelist and a user whitelist. To login successfully you need to pass each whitelist test.